unity/unity-embed-host/CoreCLRHost.cs

```diff
@@ -708,14 +690,6 @@ public static IntPtr unity_method_get_function_pointer([NativeCallbackType("Mono
         return tClass.MakeArrayType((int)rank).TypeHandleIntPtr();
     }
 
-
-    [return: NativeCallbackType("gint64")]
-    public static long gc_get_heap_size()
```

Comment:
Are you going to remove this from https://github.cds.internal.unity3d.com/unity/neutron/blob/main/External/ScriptingCore/Unity.Private.Scripting/Interop/GCInterop.cs ?  Same question for `gc_max_generation`

Reply:
No that one will stay. This change isn't probably completely necessary as I think you are removing all of the embed host stuff anyway right?

Reply:
Yes.